index.js

```diff
@@ -250,7 +250,10 @@ export function useGesture(props, config) {
       handleGesture('onDrag')
     }
 
-    const cancelDrag = event => requestAnimationFrame(() => onDragEnd(event, true))
+    const cancelDrag = event => {
+      updateState({ drag: { canceled: true } })
+      requestAnimationFrame(() => onDragEnd(event, true))
```

Comment:
The second argument to `onDragEnd` is used to set `canceled` state attribute. Since we're now updating it in `cancelDrag`, it's now useless! Can you please remove it from `onDragEnd`?